multimodal/src/autogluon/multimodal/utils/matcher.py

```diff
@@ -332,7 +337,7 @@ def semantic_search(
     response_data: Optional[Union[pd.DataFrame, dict, list]] = None,
     query_embeddings: Optional[torch.Tensor] = None,
     response_embeddings: Optional[torch.Tensor] = None,
-    query_chunk_size: int = 100,
+    query_chunk_size: int = 128,
     response_chunk_size: int = 500000,
```

Comment:
will 500000 be too large for some machines?

Reply:
sentence transformer (https://www.sbert.net/examples/applications/semantic-search/README.html#sentence_transformers.util.semantic_search) uses this number.